meson.build

```diff
@@ -21,7 +21,7 @@ libsmartcols_version = '1.1.0'
 libfdisk_version = '1.1.0'
 
 prefixdir = get_option('prefix')
-if not prefixdir.startswith('/')
+if not fs.is_absolute(fs.expanduser(prefixdir))
```

Comment:
Shouldn't the shell have expanded `~`?
It seems weird do it here.

+1 for `is_absolute`

Reply:
Yeah, I think it is kind of awkward having this as the only place `expanduser` is used. I would think it would be expanded by the shell, but it may be quoted or the option may be passed via Meson native or cross file. I can remove it if you prefer though.

Reply:
Personally I'd like to leave it out.

Reply:
Sure thing, I've updated.